### PR TITLE
Include new kickstart files for CCN profiles

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

Include three new rhel9 kickstart files for CCN profiles.

#### Rationale:

Easier to test CCN profiles.

#### Review Hints:

They were based on CIS kickstart files. I only updated the profile name at the end of the file and some comment lines.

### CODE REVIEW

File: products/rhel9/kickstart/ssg-rhel9-ccn_basic-ks.cfg [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+logvol /var/tmp --fstype=xfs --name=vartmp --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# Ensure /var Located On Separate Partition
+logvol /var --fstype=xfs --name=var --vgname=VolGroup --size=3072
+# Ensure /var/log Located On Separate Partition
+logvol /var/log --fstype=xfs --name=varlog --vgname=VolGroup --size=1024
+# Ensure /var/log/audit Located On Separate Partition
+logvol /var/log/audit --fstype=xfs --name=varlogaudit --vgname=VolGroup --size=512
+logvol swap --name=swap --vgname=VolGroup --size=2016
+
+# Harden installation with CCN profile (Advanced)
```
Comment (vojtapolasek) [inline products/rhel9/kickstart/ssg-rhel9-ccn_basic-ks.cfg:113]:
```suggestion
# Harden installation with CCN profile (Basic)
```

File: products/rhel9/kickstart/ssg-rhel9-ccn_intermediate-ks.cfg [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+logvol /var/tmp --fstype=xfs --name=vartmp --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# Ensure /var Located On Separate Partition
+logvol /var --fstype=xfs --name=var --vgname=VolGroup --size=3072
+# Ensure /var/log Located On Separate Partition
+logvol /var/log --fstype=xfs --name=varlog --vgname=VolGroup --size=1024
+# Ensure /var/log/audit Located On Separate Partition
+logvol /var/log/audit --fstype=xfs --name=varlogaudit --vgname=VolGroup --size=512
+logvol swap --name=swap --vgname=VolGroup --size=2016
+
+# Harden installation with CCN profile (Advanced)
```
Comment (vojtapolasek) [inline products/rhel9/kickstart/ssg-rhel9-ccn_intermediate-ks.cfg:113]:
```suggestion
# Harden installation with CCN profile (Intermediate)
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello @marcusburghardt  and thank you for the addition. I found just two small things in comments... no big deal but it could cause some confusion for users.

File: products/rhel9/kickstart/ssg-rhel9-ccn_basic-ks.cfg (hunk 1 above)
Comment (marcusburghardt) [reply products/rhel9/kickstart/ssg-rhel9-ccn_basic-ks.cfg:113]: Great. I missed this line when copying and pasting. Thanks

**Review by vojtapolasek — APPROVED** [review]
Looks good now, thank you.

**Pull request merged**